tests/Backend/Snowflake/TimestampTest.php

```diff
@@ -291,20 +200,40 @@ public function testTimestampCopyImport()
         ));
         unset($db);
 
-        $tmpWorkspaceInc = $workspaces->createWorkspace();
-        $tmpWorkspaceDb = $this->getDbConnection($tmpWorkspaceInc['connection']);
-        $workspaces->cloneIntoWorkspace($tmpWorkspaceInc['id'], [
+        $this->assertDataInTable($workspaces, $table['id'], 'timestamptestInc', 3);
+    }
+
+    /**
+     * @param Workspaces $workspaceApi
+     * @param string $tableId
+     * @param string $workspaceTableName
+     * @param int $expectedRows
+     */
+    private function assertDataInTable($workspaceApi, $tableId, $workspaceTableName, $expectedRows)
+    {
+        $tmpWorkspace = $workspaceApi->createWorkspace();
```

Comment:
Potřebuješ pokaždé založit nový workspace? Nestačilo by na kontrolu jen jeden v setupBeforeClass?

Reply:
Krom toho, že to stojí prachy je to i dost pomalé...

Reply:
změnil sem, kleslo to z ~8min na ~6min

Reply:
25% not bad...